src/controllers/observation-controller.ts

```diff
@@ -60,26 +51,46 @@ export default class ObservationController extends AbstractController {
   };
 
   private addObservation = async (req: Request, res: Response, next: NextFunction) => {
-    const newObservation = req.body;
-    if (!req.user) {
-      return res.status(401).send();
+    const rawObservation = req.body;
+    if (!req.user) res.status(401).send();
```

Comment:
This should be processed by auth middleware already.

Reply:
Removed, thanks for pointing out!